pkg/agents/team7/agent3/agent.go

```diff
@@ -0,0 +1,33 @@
+package team7agent3
+
+import (
+	"github.com/SOMAS2021/SOMAS2021/pkg/infra"
+)
+
+type CustomAgent1 struct {
```

Comment:
```suggestion
type CustomAgent3 struct {
```
This should be changed for all instances.